pom.xml

```diff
@@ -15,6 +15,13 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>false</reuseForks>
```

Comment:
We don't have parallel tests, so I think this option shouldn't be necessary?

Reply:
It seems that sometimes part of the spring context can be kept alive if it reuses the fork for each test group. 

http://forum.spring.io/forum/spring-projects/data/71251-embedded-datasources-are-not-dropped-when-context-is-destroyed

So it is to err on the safe side as I've seen that some combination of test execution doesn't drop the database.